win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1978,91 +2056,63 @@ IntGetOutlineTextMetrics(PFONTGDI FontGDI,
     TT_HoriHeader *pHori;
     TT_Postscript *pPost;
     FT_Fixed XScale, YScale;
-    FT_WinFNT_HeaderRec Win;
+    FT_WinFNT_HeaderRec WinFNT;
     FT_Error Error;
-    char *Cp;
-    UNICODE_STRING FamilyNameW, FaceNameW, StyleNameW, FullNameW;
+    BYTE *pb;
+    FONT_NAMES FontNames;
     PSHARED_FACE SharedFace = FontGDI->SharedFace;
-    PSHARED_FACE_CACHE Cache = (PRIMARYLANGID(gusLanguageID) == LANG_ENGLISH) ? &SharedFace->EnglishUS : &SharedFace->UserLanguage;
+    PSHARED_FACE_CACHE Cache;
     FT_Face Face = SharedFace->Face;
 
+    if (PRIMARYLANGID(gusLanguageID) == LANG_ENGLISH)
+        Cache = &SharedFace->EnglishUS;
+    else
+        Cache = &SharedFace->UserLanguage;
+
     if (Cache->OutlineRequiredSize && Size < Cache->OutlineRequiredSize)
-    {
```

Comment:
I think the general consensus is that we want braces around all blocks, so this change is going in the wrong direction.